internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```
@@ -32,6 +32,11 @@ EnvironmentManagerRole:
 {{- range $arn := .PrivateHTTPConfig.ImportedCertARNs}}
           - "{{$arn}}"
 {{- end}}
+{{- if .CDNConfig}}
+{{- if .CDNConfig.ViewerCertificate}}
```

```suggestion
{{- if and .CDNConfig .CDNConfig.ViewerCertificate}}
```

You can do this instead of two if statements